.gitignore

```diff
@@ -53,6 +53,8 @@ docs/_build/
 
 # editor files
 .idea
+.vscode
```

Comment:
I also do some front end development. How is PyCharm for it? I was anyway planning to buy a paid version when I have enough money.

Reply:
PyCharm is pretty cool also for frontend, it has a lot of advantages on code checking for js and css and autocomplete for HTML. However, there are another IDE from Jetbrains called WebStorm and it's specialist with frontend development. I believe the community version is enough and free.

Reply:
No no, I wanted to pay because I liked the work people put behind it. I like people to pay for such projects so that the opensource projects can support themselves.

Will try PyCharm in the future. Using different IDEs for front end and back end will mess more things inside my head than they are currently are. I am not sure if I can handle that :-p